tests/test_data/study_python_valid_generated.md

```diff
@@ -0,0 +1,15 @@
+## study_python_valid base tables
+
+### study_python_valid__lib_transactions
+|    Column     |  Type   |Description|
```

Comment:
I might have missed this - how does Description get filled in? Might be nice to have a test row that includes one.

Reply:
right now - by hand, after the fact. Let me add a note about this someplace.